source/clear-linux/guides/network/ovs-dpdk.rst

```diff
@@ -67,41 +78,39 @@ Using Linux Bridges
           exit 1
       fi
 
-#. Enable execute permission on the script, preserving its group permissions
-   with respect to other files.
+#. Enable execute permission on the script. Preserve the group
+   permissions with respect to other files using the options below.
 
    .. code-block:: bash
 
-      # chmod a+x qemu-ifup
+      chmod a+x qemu-ifup
 
-#. Create a bridge using the :command:`brctl` tool; you can verify whether
-   the bridge was successfully created by using ip tool.
+#. Create a bridge using the :command:`brctl` tool. Use the `ip` tool to
+   verify whether or not the bridge was successfully created.
 
    .. code-block:: bash
 
-      # brctl addbr br0
-
-   .. note::
+      brctl addbr br0
+      ip a
 
-      At this point, it is possible to add a NIC with
-      ``brctl addif br0 <network interface>``, example:
+#. Add a NIC using the syntax `brctl addif br0 <network interface>`.
 
    .. code-block:: bash
 
-      # brctl addif br0 enp3s0f0
+      brctl addif br0 enp3s0f0
```

Comment:
Where does enp3s0f0 come from?  is it an ouptut from a previous command described above?

Reply:
@MCamp859 This is network interface of the user's host system. It is displayed in the previous step "ip a"